utils/form-schema.ts

```diff
@@ -0,0 +1,24 @@
+import { z } from "zod";
+
+export const signUpSchema = z.object({
+  full_name: z.string().min(1,{message: "Name is required."}),
```

Comment:
Consider adding max length validations to prevent excessively long inputs eg.
```suggestion
  full_name: z.string().min(1, { message: "Name is required." }).max(100, { message: "Name is too long." }),
```